.prettierignore

```diff
@@ -11,3 +11,4 @@ typechain-types
 public/snippets
 public/search-index.json
 vercel.json
+src/content/quickstarts
```

Comment:
```suggestion
src/content/quickstarts/**/*.mdx
```

We could still get prettier working for `.astro` files?

Reply:
Good idea! [Done in the quickstarts PR](https://github.com/smartcontractkit/documentation/pull/1458/commits/f3b404c5f3ab6a2a7a2297e5e6cfd468a172b9f3)